oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/LuceneIndexInfoProvider.java

```diff
@@ -140,6 +139,24 @@ private static void computeLastUpdatedTime(NodeState idxState, LuceneIndexInfo i
         }
     }
 
+    private static void checkIfHiddenNodesExists(NodeState idxState, LuceneIndexInfo info) {
+        // Check for hidden oak libs mount node that has indexed content for read only repo in composite store
+        info.hasHiddenOakLibsMount = false;
+        for(String c : idxState.getChildNodeNames()) {
+            if (c.startsWith(IndexDefinition.HIDDEN_OAK_MOUNT_PREFIX)) {
```

Comment:
I am not sure about this condition here. Instead of `startsWith` shouldn't we check for a node with name `:oak:mount-libs-index-data`? cc @thomasmueller

Reply:
@fabriziofortino  - we have the same check currently in the isActiveIndex check in IndexName java - so this should work.

Reply:
`startsWith` makes sense in my view... The cases we have so far is all `:oak:mount-libs-index-data`, but someone might change the read-only mount names. I think the prefix will always stay like this.